test/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ add_dependencies(test_c ${PROJECT_NAME}_preload)
 function(set_common_properties test_name)
     set_property(TEST ${test_name} APPEND PROPERTY ENVIRONMENT DFTRACER_LOG_LEVEL=DEBUG)
     set_property(TEST ${test_name} APPEND PROPERTY ENVIRONMENT DFTRACER_TRACE_COMPRESSION=0)
-    set_property(TEST ${test_name} APPEND PROPERTY ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/${DFTRACER_LIBDIR})
+    set_property(TEST ${test_name} APPEND PROPERTY ENVIRONMENT LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/${DFTRACER_LIBDIR}:$ENV{TEST_LD_LIBRARY_PATH})
```

Comment:
Adding this (along with the line 128) to support additional TEST_LD_LIBRARY_PATH.

In ALCF Polaris, sometimes we have problem where we cannot find `libfabric.so` (our last meeting showed the problem)

Reply:
U should not add it like this. Add a option on CMake and then use it internally. Thats is less opaque as other people can use it correctly.

Reply:
fixed using `DFTRACER_TEST_LD_LIBRARY_PATH` options on cmake and modified setup.py as well